Explorer/Assets/DCL/PerformanceAndDiagnostics/Profiling/ECS/ProfilingSystem.cs

```diff
@@ -0,0 +1,107 @@
+        {
+            this.profilingProvider = profilingProvider;
+            this.memoryBudgetProvider = memoryBudgetProvider;
+
+            CreateView();
+
+            void CreateView()
+            {
+                var version = new ElementBinding<string>(Application.version);
+
+                DebugWidgetBuilder perfWidget = debugBuilder.AddWidget("Performance")
+                                                            .SetVisibilityBinding(visibilityBinding = new DebugWidgetVisibilityBinding(true))
+                                                            .AddCustomMarker("Version:", version)
+                                                            .AddCustomMarker("Frame Rate:", fps = new ElementBinding<string>(string.Empty))
+                                                            .AddMarker("Hiccups last 1000 frames:", hiccups = new ElementBinding<ulong>(0), DebugLongMarkerDef.Unit.NoFormat);
+
+                if (!Debug.isDebugBuild)
+                    perfWidget.AddCustomMarker("Total Used Memory:", usedMemory = new ElementBinding<string>(string.Empty))
+                              .AddCustomMarker("Memory Budget Thresholds:", memoryCheckpoints = new ElementBinding<string>(string.Empty));
+                else
+                    debugBuilder.AddWidget("Memory")
+                                .SetVisibilityBinding(memoryVisibilityBinding = new DebugWidgetVisibilityBinding(true))
+                                .AddCustomMarker("Total Used Memory:", usedMemory = new ElementBinding<string>(string.Empty))
+                                .AddCustomMarker("Memory Budget Thresholds:", memoryCheckpoints = new ElementBinding<string>(string.Empty))
+                                .AddSingleButton("Memory NORMAL", () => MemoryBudgetProvider.DebugMode = MemoryUsageStatus.Normal)
+                                .AddSingleButton("Memory WARNING", () => MemoryBudgetProvider.DebugMode = MemoryUsageStatus.Warning)
+                                .AddSingleButton("Memory FULL", () => MemoryBudgetProvider.DebugMode = MemoryUsageStatus.Full);
+            }
+        }
+
+        protected override void Update(float t)
+        {
+            if ((visibilityBinding.IsExpanded || memoryVisibilityBinding.IsExpanded) && lastTimeSinceMetricsUpdate > 0.5f)
+            {
+                lastTimeSinceMetricsUpdate = 0;
+                UpdateView();
+            }
+
+            lastTimeSinceMetricsUpdate += t;
+            profilingProvider.CheckHiccup();
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private void UpdateView()
+        {
+            hiccups.Value = profilingProvider.HiccupCountInBuffer;
+            usedMemory.Value = $"<color={GetMemoryUsageColor()}>{profilingProvider.TotalUsedMemoryInMB}</color> MB";
```

Comment:
You don't need a separate property with `MB`: just use `BytesFormatter.Normalize(TotalUsedMemoryInBytes)`. Also change `long` to `ulong` please